### PR TITLE
Avoid layout shift 

### PR DESCRIPTION
**Pull request by ap-justin**
## Explanation of the solution
* set initial content height to a minimum of `100dvh - headerHeight`
```
-----------|-- screen start
header     |
-----------|
           |
content    |
           |
-----------|-- screen end
footer     |


```

footer would always be the next thing the user see when first scrolling (unless content is longer than what is initially set)
image

## Instructions on making this work

- run `yarn` or `yarn install` to install npm dependencies
- run `yarn run test --watchAll` to verify all tests still pass
- (optional) run `yarn run build` to verify the build passes
- run `yarn start` to start the webapp
-

## UI changes for review

When major UI changes are introduced with a PR, please include links to URLS to compare or screenshots demonstrating the difference and notify on design changes

### CODE REVIEW

File: src/App/Header/index.tsx [hunk 1]
```diff
@@ -6,26 +6,24 @@ import NavDropdown from "./NavDropdown";
...
 
-export default function Header({ links }: Props) {
+export default function Header({ links, classes }: Props) {
   const location = useLocation();
-  const headerClassNames = useHeaderClassNames();
+  const headerClassNames = useHeaderClassNames(classes);
 
   return (
     <header className={headerClassNames}>
-      <div className="px-0 md:py-2 h-full w-full bg-white">
```
Comment (ap-justin) [inline src/App/Header/index.tsx:17]: remove extra div

File: src/App/Layout.tsx [hunk 2]
```diff
@@ -12,9 +12,9 @@ const { GROUPS_DATA, SOCIAL_MEDIA_LINKS } = CHARITY_LINKS;
 export default function Layout() {
   const headerLinks = useHeaderLinks();
   return (
-    <div className="grid grid-rows-[auto_1fr_auto]">
+    <div className="grid grid-rows-[4rem_minmax(calc(100dvh-4rem),1fr)_auto]">
       <Seo /> {/* Load all defaults for SEO meta tags */}
-      <Header links={headerLinks} /> {/* sticky component, not part of grid */}
+      <Header links={headerLinks} classes="sticky top-0 z-20" />
```
Comment (ap-justin) [inline src/App/Layout.tsx:17]: set outside classes here

File: src/index.html [hunk 3]
```diff
@@ -5,7 +5,7 @@
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
     <link href="https://fonts.googleapis.com/css2?family=DM+Sans:ital,opsz,wght@0,9..40,400;0,9..40,500;0,9..40,600;0,9..40,700;0,9..40,800;1,9..40,400&family=Quicksand:wght@400;500;600;700&display=swap" rel="stylesheet">
   </head>
-  <body class="grid min-h-screen">
+  <body class="grid min-h-dvh">
```
Comment (ap-justin) [inline src/index.html:8]: [`dvh`](https://tailwindcss.com/blog/tailwindcss-v3-4#dynamic-viewport-units) is shipped in tailwind 3.4

Comment (SovereignAndrey) [reply src/index.html:8]: That's a useful class. :+1:

**Review by SovereignAndrey — APPROVED** [review]

**Pull request merged**